src/metabase/driver.clj

```diff
@@ -901,8 +901,9 @@
   :hierarchy #'hierarchy)
 
 (defmulti create-table!
-  "Create a table named `table-name`. If the table already exists it will throw an error."
-  {:added "0.47.0", :arglists '([driver db-id table-name col->type])}
+  "Create a table named `table-name`. If the table already exists it will throw an error.
+  `primary-key` is optional and is a vector of column names that will be used as the primary key."
+  {:added "0.47.0", :arglists '([driver database-id table-name column->type primary-key])}
```

Comment:
Is it not worth future proofing things somewhat and making the last parameter a metadata map or kwargs? I could imagine indexes for example becoming important if we make imports more scalable.